src/current/_includes/releases/v23.2/v23.2.0-alpha.1.md

```diff
@@ -158,6 +158,17 @@ Release Date: September 23, 2023
 - There is now a `CREATEDB` system privilege, which is analogous to the existing `CREATEDB` role option, but can also be inherited by role membership. [#110220][#110220]
 - There is now a `CONTROLJOB` system privilege, which is analogous to the existing `CONTROLJOB` role option, but can also be inherited by role membership. [#110220][#110220]
 - The `persistedsqlstats` table maximum size check is now done once an hour instead of every 10 minutes. This reduces the risk of serialization errors on the statistics tables. [#110173][#110173]
+- The new [cluster setting](https://www.cockroachlabs.com/docs/{{ page.version.version }}/cluster-settings) `sql.txn.read_committed_syntax.enabled`, controls whether transactions run under `READ COMMITTED` or `SERIALIZABLE` isolation. It defaults to false. When set to true, the following statements will configure transactions to run under READ COMMITTED isolation:
```

Comment:
Why are we using the full link here? prior to the liquid? (Sorry if I missed something about this.)

Reply:
@kathancox The full link will make less work for me later on when we split the releases into its own Netlify site. We'll need to use full links between each docs page so that htmltest doesn't break.

Aside from that, the link will fail htmltest anyway, because of the Liquid. Proposing this instead:

```suggestion
- The new [cluster setting](https://www.cockroachlabs.com/docs/v23.2/cluster-settings) `sql.txn.read_committed_syntax.enabled`, controls whether transactions run under `READ COMMITTED` or `SERIALIZABLE` isolation. It defaults to false. When set to true, the following statements will configure transactions to run under READ COMMITTED isolation:
```